ios/browser/api/ads/brave_ads.mm

```diff
@@ -269,10 +268,6 @@ - (void)shutdownService:(nullable void (^)())completion {
           // Deprecate shutdown API call.
           self->ads->Shutdown(base::BindOnce(^(bool) {
             [self deallocAds];
-            [self deallocAdsClientNotifier];
```

Comment:
now we should dealloc AdsClientNotifier only during BraveAds dealloc, because we create it during init call.